packages/mwp-app-render/src/components/GoogleTagManager.jsx

```diff
@@ -0,0 +1,76 @@
+// @flow
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// Runtime flag
+export const GOOGLE_TAG_MANAGER_RTF = 'google_tag_manager';
+
+// Production and dev keys
+const GTM_KEYS = {
+	'prod': 'GTM-T2LNGD', 
+	'dev': 'GTM-W9W847',
+};
+
+/*
+ * @description Gets google tag manager script with key applied
+ * @see {@link https://developers.google.com/tag-manager/quickstart}
+*/
+const getScript = (key: string) => {
+	const script = `(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','${key}');` 
+	return <script>{script}</script>
```

Comment:
2 things to watch out for:

1. This might need to be `<script dangerouslySetInnerHtml={ { _html: script } } />` in order to avoid HTML escaping the contents
2. I like the line breaks that are used in https://github.com/holidaycheck/react-google-tag-manager/blob/master/src/build_parts.js - even though it's minified, you can at least see the whole script without side scrolling

Reply:
re: 1. Gotcha. Thanks.
re: 2. Yeah, I also like that. Will update.